README.md

```diff
@@ -167,10 +167,20 @@ $ ninja -C build && ( cd build ; lit test )
 - `TRITON_INTERPRET=1` uses the Triton interpreter instead of running on the
   GPU.  You can insert Python breakpoints in your kernel code!
 - `TRITON_ENABLE_LLVM_DEBUG=1` passes `-debug` to LLVM, printing a lot of
-  debugging information to stdout.  If this is too noisy, run with just
+  debugging information to stdout. If this is too noisy, please try
+  `TRITON_LLVM_DEBUG_ONLY` instead to limit the output.
+
+  The other alternative to reduce output noisiness is running with just
   `LLVM_IR_ENABLE_DUMP=1`, extract the IR before the LLVM pass of interest, and
   then run LLVM's `opt` standalone, perhaps passing `-debug-only=foo` on the
   command line.
+- `TRITON_LLVM_DEBUG_ONLY=<comma separated>` is the equivalent of LLVM's
+  `-debug-only` command line option. This limits the LLVM debug output to
+  specific pass names or components of LLVM in order to allow the debug output
+  to be less noisy. `TRITON_LLVM_DEBUG_ONLY` allows for one or more comma
+  separated pass names to be specified (ie
```

Comment:
above we said they were pass names or components, but here we say they're pass names, that's a bit confusing.  Instead we can say "one or more comma-separated values", since we already said what the values are